boot/zephyr/pm.yml

```diff
@@ -24,7 +33,6 @@ mcuboot_secondary:
     align: {start: 4}
 #else
   placement:
-    align: {start: CONFIG_FPROTECT_BLOCK_SIZE}
```

Comment:
Should we also remove the `align_next`? Should we consider aligning to size of the FLASH page that is the smallest chunk of memory which could be erased separately?

Reply:
> Should we also remove the `align_next`? 

Do we need that? We need the start of that to be aligned with FPROTECT granularity, otherwise we cannot safely separate them.

> Should we consider aligning to size of the FLASH page that is the smallest chunk of memory which could be erased separately?

Do you mean the alignment of the `align_next` ? Or the `swap area` ?

Reply:
> Do we need that? We need the start of that to be aligned with FPROTECT granularity, otherwise we cannot safely separate them.

If we want to align the partition to FPROTECT granularity, shouldn't we also align the partition start here? Or it's enough that the alignment is added under `mcuboot_secondary_pad`?

Reply:
> Do you mean the alignment of the `align_next` ? Or the `swap area` ?
I was thinking about ensuring that image slots' size is multiple of `CONFIG_FLASH_LAYOUT_PAGE_SIZE` to ensure that we can erase it safely without interfering with other data in FLASH (if we place the image data in internal FLASH).